whatwg.org/chat

```diff
@@ -27,38 +27,42 @@
  <a href="https://participate.whatwg.org/">Participate</a>
 </nav>
 
-<h3 id="irc">IRC<a class="self-link" href="#irc"></a></h3>
+<h3 id="chat">Chat<a class="self-link" href="#irc"></a></h3>
 
-<p>The Freenode IRC network has a channel called <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> where some of the more active WHATWG community members hang out. Feel free to join us!</p>
+<p>We have both a <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> channel on the Freenode IRC network and a <a href="https://app.element.io/#/room/#whatwg:matrix.org">whatwg</a> room in the <a href="https://en.wikipedia.org/wiki/Matrix_(protocol)">Matrix</a> network. Join us!</p>
 
-<p>Note that if you go to IRC to ask a question, it might take a while to get a reply. It can pay off to stick around for a couple of hours or more.</p>
+<p>Note that if you ask a question in either of those places, it might take a while to get a reply. It can pay off to stick around for a couple of hours or more.</p>
 
-<p>If you want to run a bot, let us know. If they are useful, e.g. providing logging facilities, then they are more than welcome.</p>
+<h3 id="getting-started">Getting Started with IRC or Matrix<a class="self-link" href="#getting-started"></a></h3>
+
+<p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there. In order to join the channel, you will need to <a href="https://freenode.net/kb/answer/registration">register your nick on Freenode</a>.</p>
```

Comment:
I think the positioning of this implies that the simplest way to get started _is_ using IRC, but I don't think that's true; I'd prefer we direct new users to the service that doesn't require magic incantations to avoid being spammed etc.

Generally if the plan is to have both FreeNode and Matrix on an ongoing basis, I'd suggest that we divide this section up into two subsections for the different options (I'd also prefer we just make a decision instead of having two options, but I see why you chose this route for the PR).

Reply:
> I think the positioning of this implies that the simplest way to get started _is_ using IRC, but I don't think that's true; I'd prefer we direct new users to the service that doesn't require magic incantations to avoid being spammed etc.

OK, ec88c16 is an attempt to reposition things along those lines.

> Generally if the plan is to have both FreeNode and Matrix on an ongoing basis, I'd suggest that we divide this section up into two subsections for the different options

OK, I’ll put together another commit which does that